pysat/tests/test_utils_io.py

```diff
@@ -190,6 +190,40 @@ def test_basic_write_and_read_netcdf_mixed_case_meta_format(self):
 
         return
 
+    @pytest.mark.parametrize("kwargs,target",
+                             [({}, True),
+                              ({'export_pysat_info': True}, True),
+                              ({'export_pysat_info': False}, False)])
+    def test_basic_write_and_read_netcdf_export_pysat_info(self, kwargs,
+                                                           target):
+        """Test basic netCDF4 read/write with optional pysat info export.
+
+        Parameters
+        ----------
+        kwargs : dict
+            Specify value of `export_pysat_info`. An empty dict sets to
+            default value.
+        target : bool
+            True is pysat info is expected to be written to file.
+        """
+        # Create a bunch of files by year and doy
+        outfile = os.path.join(self.tempdir.name, 'pysat_test_ncdf.nc')
+        self.testInst.load(date=self.stime, use_header=True)
+
+        io.inst_to_netcdf(self.testInst, fname=outfile, preserve_meta_case=True,
+                          epoch_name=default_epoch_name, **kwargs)
+
+        tkwargs = decode_times_val(self.testInst.pandas_format)
+
+        self.loaded_inst, meta = io.load_netcdf(
```

Comment:
I'm not sure what is meant here.  Is this a change to `load_netcdf` behaviour, or a change in the test?

Reply:
This would change the `load_netcdf` behaviour.  This may or may not be desired, but it would make the instrument look more like the original instrument that created the file.

Reply:
I think this is a design discussion for a future meeting.